nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java

```diff
@@ -92,6 +93,8 @@
 
     protected static final String THREAD_PREFIX = "Nuxeo-Work-";
 
+    public static final String SHUTDOWN_DELAY_KEY = "nuxeo.work.shutdown.delay";
```

Comment:
Please add javadoc with `@since 10.2` and a description about time unit.

Reply:
I am in favor of adding the unit to the key: `nuxeo.work.shutdown.delay.ms` this makes things clear, don't expect a sysadmin to read a javadoc to configure a framework.

Reply:
Would it be possible to use seconds instead ? We're gonna make this unit a nuxeo standard.

Reply:
Let's keep ms if it's accurate in this case.

Reply:
no time unit sucks